### DISCUSSION TITLE
Wrap node text

### DISCUSSION THREAD

**Discussion by SethAlilio** [joyceworks/flowchart-react#28] [Q&A] [answered]
How can I wrap the text so it wouldn't overflow the node, something like this:
![image](https://user-images.githubusercontent.com/30690066/206978926-05815b11-b83c-4820-93f7-44c4a22ee792.png)

**Comment by iamppz** [comment]
Hello, thank you for the feedback, I will work on it later.

**Comment by iamppz** [comment, marked as answer]
Hi, Please upgrade flowchart-react to 3.9.0.

I'm so sorry that it took so long because I got COVID last week.

**Reply by SethAlilio** [reply]
No worries, thank you so much, hope you're okay now
